src/cascadia/LocalTests_SettingsModel/ColorSchemeTests.cpp

```diff
@@ -290,4 +291,81 @@ namespace SettingsModelLocalTests
             VERIFY_ARE_EQUAL(ARGB(0, 7, 7, 7), scheme2->_Background);
         }
     }
+
+    void ColorSchemeTests::UpdateSchemeReferences()
+    {
+        const std::string settingsString{ R"({
+                                                "defaultProfile": "Inherited reference",
+                                                "profiles": {
+                                                    "defaults": {
+                                                        "colorScheme": "Scheme 1"
+                                                    },
+                                                    "list": [
+                                                        {
+                                                            "name": "Explicit scheme reference",
+                                                            "colorScheme": "Scheme 1"
+                                                        },
+                                                        {
+                                                            "name": "Explicit reference; hidden",
+                                                            "colorScheme": "Scheme 1",
+                                                            "hidden": true
+                                                        },
+                                                        {
+                                                            "name": "Inherited reference"
+                                                        },
+                                                        {
+                                                            "name": "Different reference",
+                                                            "colorScheme": "Scheme 2"
+                                                        }
+                                                    ]
+                                                },
+                                                "schemes": [
+                                                    { "name": "Scheme 1" },
+                                                    { "name": "Scheme 2" },
+                                                    { "name": "Scheme 1 (renamed) " }

```

Comment:
@DHowett dumb question:
```suggestion
                                                    { "name": "Scheme 1 (renamed)" }
```
this doesn't work because the syntax expects `)"` to be the end of the string. But I've tried doing `)/"` and `)\"` and both seem to be interpreted incorrectly. Any ideas how to get around this?

Reply:
That's why I went with the extra space haha

Reply:
Use a different delimiter. The [raw string literal](https://en.cppreference.com/w/cpp/language/string_literal) syntax supports different delimiters... it doesn't just have to be `R"(...)"`, it can be `R"json(xxxxxxxxxxxxxx)json"`